docs/operators/from_azure_blob_storage.md

```diff
@@ -0,0 +1,134 @@
+---
+title: from_azure_blob_storage
+category: Inputs/Events
+example: 'from_azure_blob_storage "abfs://container/data/**.json"'
+---
+
+Reads one or multiple files from Azure Blob Storage.
+
+```tql
+from_azure_blob_storage url:string, [account_key=string, watch=bool, remove=bool, rename=string->string, path_field=field] { … }
+```
+
+## Description
+
+The `from_azure_blob_storage` operator reads files from Azure Blob Storage,
+with support for glob patterns, automatic format detection, and file
+monitoring.
+
+By default, authentication is handled by the Azure SDK's credential chain which
+may read from multiple environment variables, such as:
+
+- `AZURE_TENANT_ID`
+- `AZURE_CLIENT_ID`
+- `AZURE_CLIENT_SECRET`
+- `AZURE_AUTHORITY_HOST`
+- `AZURE_CLIENT_CERTIFICATE_PATH`
+- `AZURE_FEDERATED_TOKEN_FILE`
+
+### `url: string`
+
+URL identifying the Azure Blob Storage location where data should be read from.
+
+The characters `*` and `**` have a special meaning. `*` matches everything
+except `/`. `**` matches everything including `/`. The sequence `/**/` can also
+match nothing. For example, `container/**/data` matches `container/data`.
+
+Supported URI formats:
+
+1. `abfs[s]://[:<password>@]<account>.blob.core.windows.net[/<container>[/<path>]]`
+2. `abfs[s]://<container>[:<password>]@<account>.dfs.core.windows.net[/path]`
+3. `abfs[s]://[<account[:<password>]@]<host[.domain]>[<:port>][/<container>[/path]]`
+4. `abfs[s]://[<account[:<password>]@]<container>[/path]`
```

Comment:
This is not true anymore, see `azurefs.h`:

>   /// 1. abfs[s]://\<account\>.blob.core.windows.net[/\<container\>[/\<path\>]]
  /// 2. abfs[s]://\<container\>\@\<account\>.dfs.core.windows.net[/path]
  /// 3. abfs[s]://[\<account@]\<host[.domain]\>[\<:port\>][/\<container\>[/path]]
  /// 4. abfs[s]://[\<account@]\<container\>[/path]